FutureNorthants/northants-design-system
# Update storybook to 8.1.6

Update to latest version of storybook to remove ip dependency. 

## Testing
- Checkout this branch and run `npm install`
- Run tests with `npm run test`
- Run `npm run dev` to manually test updates
- Run `yalc publish` to ensure it builds and then in the frontend run `yalc add northants-design-system && npm install && npm run ct` to run component tests.
- In the frontend, run `npm run test` to run the integration tests
- In the frontend run ` npm run build` to test site builds
